src/webgpu/shader/execution/expression/call/builtin/textureLoad.spec.ts

```diff
@@ -271,14 +392,73 @@ Parameters:
   )
   .params(u =>
     u
+      .combine('format', kTestableColorFormats)
+      // MAINTENANCE_TODO: Update createTextureFromTexelViews to support depth32float and remove this filter.
```

Comment:
This should be a regular `TODO` in the test description since it impacts test coverage.

Reply:
A TODO for adding depth formats is already at the top

Reply:
Got it, LGTM